File: gl2/hostgl2/sqiral.f
cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
c									 c
c 		 Copyright (C) 1983, Silicon Graphics, Inc.		 c
c									 c
c  These coded instructions, statements, and computer programs  contain  c
c  unpublished  proprietary  information of Silicon Graphics, Inc., and  c
c  are protected by Federal copyright law.  They  may  not be disclosed  c
c  to  third  parties  or copied or duplicated in any form, in whole or  c
c  in part, without the prior written consent of Silicon Graphics, Inc.  c
c									 c
cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

$	INCLUDE fgl.h

	integer*4 j

	call ginit
	call cursof
	call color(BLACK)
	call clear
	call color(BLUE)

c	move to center of the screen 
	call move2i(XMAXSC/2,YMAXSC/2)

	do 100 j=1,499,5
	    if (mod(j,2).ne.0) goto 200
		call rdr2i(j,0)
		call rdr2i(0,j)
		goto 100
200	    continue
		call rdr2i(-j,0)
		call rdr2i(0,-j)
100	continue

	call gexit
	stop
	end

File: gl2/hostgl2/sqiral.c
/**************************************************************************
 *									  *
 * 		 Copyright (C) 1983, Silicon Graphics, Inc.		  *
 *									  *
 *  These coded instructions, statements, and computer programs  contain  *
 *  unpublished  proprietary  information of Silicon Graphics, Inc., and  *
 *  are protected by Federal copyright law.  They  may  not be disclosed  *
 *  to  third  parties  or copied or duplicated in any form, in whole or  *
 *  in part, without the prior written consent of Silicon Graphics, Inc.  *
 *									  *
 **************************************************************************/

#include "gl.h"
#define even(A) (!(A%2))

main()
{
	int j;
	ginit();
	cursoff();
	color(BLACK);
	clear();
	color(BLUE);

	/* move to center of the screen */
	move2i(XMAXSCREEN/2,YMAXSCREEN/2);
	for (j=1; j<500; j=j+5) {
		if (even(j)) {
		    rdr2i(j,0);
		    rdr2i(0,j);
		}
		else  {
		    rdr2i(-j,0);
		    rdr2i(0,-j);
		}
	}
	gexit();
}

